### DISCUSSION TITLE
CentOS Stream 9 Install

### DISCUSSION THREAD

**Discussion by pappastech** [modoboa/modoboa#2641] [Installer]
Is there a timeline for an upgrade to modoboa and the modoboa installer so that it works on CentOS Stream 9?

**Comment by Spitfireap** [comment]
Good idea :), I'll try to work on it.

**Reply by pappastech** [reply]
For what it's worth, I started working on the installer and managed to get something that installed but doesn't work yet.

https://github.com/pappastech/modoboa-installer-centos9stream
